### ISSUE TITLE
DECISION TOOLS SPRINT 53 PRIORITIES Jul 28 - Aug 10

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#27997]
<hr>

## 🏖️ Sprint 53 Vacation/PTO/Holidays
<hr>

### 1. FIND A VA FORM PDF DOWNLOAD INSTRUCTIONS, EPIC #26816
- [x] Cindy -  #27480
- [x] Team -  #27998 
- [ ] Cindy -  #27479
- [x] John - Add batch 1 content to CMS - status being tracked in #22726
*Notetaking is already covered*
- Next step would be to do a survey of some type.

<hr>

### 2. DISCHARGE UPGRADE WIZARD CONTENT UPDATES - EPIC #27212
- [x] (Cassandra & Zach) [Discovery] #28002 - Create Mural of Discharge Upgrade Wizard impact to user flow based on info in #27143
- [ ] (John & Marci) - Meet with Margarete, Kayla, John S to review user flow and determine if it is an acceptable user experience
- [x] (Marci) - Collab kickoff for changes if we are still moving forward
- [ ] (Marci) - Content request for DUW changes if we are still moving forward

<hr>

### 3. SEARCH UI CONSISTENCY, EPIC #21055 
- [ ] (Marci)  **Search UI Consistency** - Document design pattern #24597 ❓ 

<hr>

### 4. TECH DEBT
- [x] (Cassandra) - #28016
- [x] (Marci) - Complete & submit Spanish translation request for "Go to online tool" verbiage (blocker for accessibility issue from Staging review) - **blocked by finalizing verbiage following mockup #28016**
- [x] (Zach) #26353
- [x] (Zach) #26904 
- [x] (Zach) #26906

### Form API Keywords/Abbreviations
- [x] Mark - Add abbreviations provided by us and VA stakeholders
- [ ] Marci - test Mark's changes on Staging

###  🗨️  Discussion Topics/Grooming 
- [ ] Well-groomed issues for Yellow Ribbon discussion
- [ ] Topic
- [ ] Topic

**Comment by cindymerrill** [comment]
Shouldn't my Synthesis and Reporting ticket also be a priority for this sprint, @MarciMcGuireGCIO ?

**Comment by MarciMcGuireGCIO** [comment]
@cindymerrill I wasn't sure which sprint to include it in.  I will move it.

**Comment by MarciMcGuireGCIO** [comment]
2021-07-30

- Cassandra - heads down on new content page; mocking up in Sketch.  Shifting to mocking the rest of the pages according to the flow
- Will pivot to working with Zach on Discharge Upgrade Wizard and mocking up updated "Go to Online Tool" link
- Zach has attended user sessions, worked on Discharge Upgrade Wizard logic mapping

**Comment by MarciMcGuireGCIO** [comment]
2021-08-02

- Zach needs Yana to look into Mural so he can wrap up his diagram
- He wanted to have the flowchart built out before he hands off to Cassandra
- Cassandra is working on HT, but will work on changing copy on online tool action links

**Comment by MarciMcGuireGCIO** [comment]
2021-08-03

- Cassandra has completed the mockup for the updated action link copy on Find a VA Form
- Zach and Cassandra are meeting to talk through the flowchart for Discharge Upgrade Wizard.
